src/Themes/ui/Theme.tsx

```diff
@@ -419,7 +420,21 @@ export function refreshTheme(): void {
   });
 
   document.body.style.backgroundColor = theme.colors.backgroundprimary?.toString() ?? "black";
+
+  const styleSheet =
+    ":root {" +
+    Object.entries(Settings.theme)
+      .map(([k, v]) => `--bb-theme-${k}: ${v}`)
+      .join(";") +
+    "}";
+
+  //The test suite does not polyfill this property
+  themeStyleSheet.replaceSync?.(styleSheet);
 }
+
+//The test suite does not polyfill this property
+document.adoptedStyleSheets?.push(themeStyleSheet);
```

Comment:
This should probably go after the refreshTheme, so that the added stylesheet is already in its proper state. Just a minor performance thing, potentially.

Reply:
Yes, it shouldn't matter in practice, aside from maybe a minor performance thing and logically being clearer.